src/blockchain/electrum.rs

```diff
@@ -325,3 +327,29 @@ crate::bdk_blockchain_tests! {
         ElectrumBlockchain::from(Client::new(&test_client.electrsd.electrum_url).unwrap())
     }
 }
+
+#[cfg(test)]
+#[cfg(feature = "test-electrum")]
+mod test {
```

Comment:
This test can be converged with the previous `bdk_blockchain_tests!` macro in a single `mod test`..